ffmpeg_progress_yield/ffmpeg_progress_yield.py

```diff
@@ -100,7 +83,7 @@ def run_command_with_progress(self, popen_kwargs={}) -> Generator[int, None, Non
                 continue
 
             stderr_line = (
-                self.process.stdout.readline().decode("utf-8", errors="replace").strip()  # type: ignore
+                self.process.stdout.readline().decode("utf-8", errors="replace").strip()
```

Comment:
This is interesting, do you know why this might have raised a type error for me? (Same with passing "q" as raw stdin.) I don't see the type error on your branch anymore.